content/docs/guides/self-hosted/api.md

```diff
@@ -38,7 +38,7 @@ The Pulumi API is one of the components required for self-hosting Pulumi in your
 
 ## What's In The Container?
 
-The API service is a Go-based application. This is a single binary application that has all of the dependencies it needs in order to run.
+The API service is a Go-based application. This is a single binary application that has all of the dependencies it needs in order to run. This container will serve using port `8080` over HTTP by default or port `8443` over HTTPS when [TLS](#tls-environment-variables) is configured.
```

Comment:
How about we create a new sub-heading so that it is easy to notice this important detail? I am thinking something like `### Ports`? If you have a better name, definitely please feel free to use that.

Reply:
Agreed, I think a separate heading would give it more visibility.